### PR TITLE
ecl 15.2.21

### PR DESCRIPTION
**Pull request by qinix**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Any chance of a `test do` block for this? Thanks :heart:

**Comment by qinix** [comment]
having a `test do` block now

File: Library/Formula/ecl.rb [hunk 1]
```diff
@@ -13,4 +13,13 @@ def install
...
     system "make install"
   end
+
+  test do
+    (testpath/'simple.cl').write <<-EOS.undent
+      (write-line (write-to-string (+ 2 2)))
+    EOS
+    output = `'#{bin}/ecl' -shell #{testpath}/simple.cl`
+    assert_equal '4', output.strip
+    assert_equal 0, $?.exitstatus
```
Comment (mikemcquaid) [inline Library/Formula/ecl.rb]: `assert_equal "4\n", shell_output("#{bin}/ecl -shell #{testpath}/simple.cl")` will do the same as the above three lines.

Comment (qinix) [inline Library/Formula/ecl.rb]: Thank you, fixed

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**